donate/templates/pages/core/ways_to_give_page_master.html

```diff
@@ -129,16 +129,16 @@ <h2>{% trans "Currencies" %}</h2>
             {% endfor %}
         </ul>
     {% endblock %}
-    {% block bitcoin %}
-        <h2>{% trans "Bitcoin" %}</h2>
+    {% block cryptocurrencies %}
+        <h2>{% trans "cryptocurrencies" %}</h2>
```

Comment:
```suggestion
        <h2>{% trans "Cryptocurrencies" %}</h2>
```

Just double checking if this should be capitalized!